requirements.txt

```diff
@@ -2,4 +2,5 @@ Mako
 PyYAML
 six
 pbr
-defusedxml
+# For OpenNebula provider
+# defusedxml
```

Comment:
OK, in fact with on `defusedxml [ON]` in the requirements.txt `pip install .'[ON]'` seems to work, I will use it is a nice way to provide alternatives.